src/sentry/static/sentry/app/views/organizationAuditLog.jsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import {History} from 'react-router';
+
+import ApiMixin from '../mixins/apiMixin';
+import DateTime from '../components/dateTime';
+import LoadingIndicator from '../components/loadingIndicator';
+import LoadingError from '../components/loadingError';
+import OrganizationHomeContainer from '../components/organizations/homeContainer';
+import OrganizationState from '../mixins/organizationState';
+import Pagination from '../components/pagination';
+import SelectInput from '../components/selectInput';
+
+import {t} from '../locale';
+
+const EVENT_TYPES = [
+  'member.invite',
+  'member.add',
+  'member.accept-invite',
+  'member.remove',
+  'member.edit',
+  'member.join-team',
+  'member.leave-team',
+  'team.create',
+  'team.edit',
+  'team.remove',
+  'project.create',
+  'project.edit',
+  'project.remove',
+  'project.set-public',
+  'project.set-private',
+  'org.create',
+  'org.edit',
+  'org.remove',
+  'tagkey.remove',
+  'projectkey.create',
+  'projectkey.edit',
+  'projectkey.remove',
+  'projectkey.enable',
+  'projectkey.disable',
+  'sso.enable',
+  'sso.disable',
+  'sso.edit',
+  'sso-identity.link',
+  'api-key.create',
+  'api-key.edit',
+  'api-key.remove'
+];
+
+
+const OrganizationAuditLog = React.createClass({
+  mixins: [
+    ApiMixin,
+    History,
+    OrganizationState,
+  ],
+
+  getInitialState() {
+    return {
+      loading: true,
+      error: false,
+      entryList: [],
+    };
+  },
+
+  componentWillMount() {
+    this.fetchData();
+  },
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.location.search !== this.props.location.search ||
+        nextProps.params.orgId !== this.props.params.orgId) {
+      this.remountComponent();
```

Comment:
@benvinegar will probably be mad about this, but I stand by the simplicity of it :)